model_test_attention2d

```diff
@@ -0,0 +1,86 @@
+import ivadomed.models as imed_model
```

Comment:
why duplicating the file and putting it under the root of the package? this is where the file should go: https://github.com/ivadomed/ivadomed/blob/master/testing/test_model.py

Reply:
sorry for that, I changed it into the test directory of the patch-1 branch, where the update_model is located.

Reply:
c3b400e doesn't solve the problem: the file is now duplicated under the testing/ folder (!)
- https://github.com/ivadomed/ivadomed/pull/552/files?file-filters%5B%5D=No+extension#diff-74343adf7b5702ca368379078d5115777493c2e5dab9ab313421d6e191e052b9
- https://github.com/ivadomed/ivadomed/pull/552/files?file-filters%5B%5D=No+extension#diff-f32ddf02dd4be1853865f9a7662cd1681e02f844066be5e3e4740b9d79b32589

Please read how "pytest" works and make the appropriate changes